src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -258,7 +258,7 @@ SGVector<T> from_gpu(const SGVector<T>& vec)
 			SG_SERROR("Data memory on GPU but no GPU backend registered. \
 						This can happen if the GPU backend was de-activated \
 						after memory has been transferred to GPU.\n");
-			return NULL;
+			return false;
```

Comment:
what do you mean not achievable? i mean that during your linalg refactor, have you had similar cases? could you maybe go through the changes pretty please? :)